### ISSUE TITLE
“best practices for busy builders” page

### ISSUE THREAD

**Issue by lidel** [ipfs/ipfs-docs#491]
We have some features which are disabled or are not defaults yet, but we encourage people to use them.
Having a single link with a list of such suggestions for busy builders could be handy

Topic ideas:

- use cidv1 for future-proof addressing, and use in case-insensitive contexts
- enable pubsub if you need fast IPNS
- enable GC if your data churn is expected to be high
- use subdomain gateways or DNSLink when publishing apps for secure context and origin isolation

**Comment by Annamarie2019** [comment]
@johnnymatthews  I'll take this one. I think this is the last of the IPFS "good first issues," so I might as well finish them up, before I move on to others.

**Comment by RuniteOre** [comment]
@Annamarie2019 Pubsub can provide faster IPNS by pushing records over it. The first publish would still take a long time because you would need to advertise the topic into the DHT, but it will happen much faster.

**Comment by lidel** [comment]
Yep, "IPNS over Pubsub" can be enabled via CLI flag (legacy way) or config (current best practice): 
https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#ipns-pubsub

We don't have it enabled by default because we have no mechanism for unsubscribing from topics that were not used for a while.  There is a tracking issue for remaining work: https://github.com/ipfs/go-ipfs/issues/8591

**Comment by Annamarie2019** [comment]
@lidel One more question: For the `ipfs config --json Ipns.UsePubsub true`, you left the link to `ipfs pubsub`  [ipfs pubsub](../reference/cli/#ipfs-pubsub) for its subcommands. I'm thinking that that's no longer relevant, or is it? Please confirm and I'll take it out. 
(BTW, we're having a little trouble reconciling the changes in the PR, so I may need to create a new PR, but I've saved yours and Johnny's suggestions and the rest of the text.)
